src/lib/style/components/query_filter/_facet-filter-header.scss

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
There are a few cases where we have two components style sheets in the same scss file, we have facet-filter-header in [_query-filter.scss](https://github.com/Sage-Bionetworks/Synapse-React-Client/blob/5db3a0093262cfebc788ad21dbed922c234e735e/src/lib/style/components/_query-filter.scss#L80), but since you now have it in its own file (which is preferable) then I think we should take its scss from _query-filter.scss and put it in this file.

Reply:
Yes, I created the new /query-filter/_facet-filter-header.scss file to match the existing typescript folder structure so it's easier to find the component css code. Like you said, it's preferable each component has its own css file unless the styles are shared among different components. I will move the _facet-filter-header specific styles to its own file.